cli/admin/src/main/java/io/pravega/cli/admin/segmentstore/FlushToStorageCommand.java

```diff
@@ -46,6 +47,7 @@ public class FlushToStorageCommand extends ContainerCommand {
 
     private static final int REQUEST_TIMEOUT_SECONDS = 60 * 30;
     private static final String ALL_CONTAINERS = "all";
+    private boolean useAdminPort = true;
```

Comment:
rename it to something more descriptive like `useFixedAdminPort`

Reply:
updated